.licenserc.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+header:
```

Comment:
Should we cover [.vib](https://github.com/bitnami/charts/tree/main/.vib) and [template](https://github.com/bitnami/charts/tree/main/template) in this PR or in a follow-up?

Reply:
I agree we should double-check several other files, but we're limiting the scope to YAML files (`Chart.yaml`, `values.yaml` and Helm templates) in this first iteration.